ports/stm32/mpconfigport.h

```diff
@@ -193,6 +193,10 @@ extern const struct _mp_obj_type_t network_lan_type;
 #define MICROPY_HW_NIC_ETH
 #endif
 
+#ifndef MICROPY_HW_ETH_DMA_ATTRIBUTE
+#define MICROPY_HW_ETH_DMA_ATTRIBUTE __attribute__((aligned(16384)));
+#endif
```

Comment:
I think this would be better in `mpconfigboard_common.h`, near the bottom of that file.

Reply:
Done